### DISCUSSION TITLE
Introducing the Cluster API Provider for Azure (CAPZ) for Kubernetes cluster management (MSFT Open Source Blog)

### DISCUSSION THREAD

**Discussion by devigned** [kubernetes-sigs/cluster-api-provider-azure#1079] [General]
Thought this might be a good place to spark any discussion leading from https://cloudblogs.microsoft.com/opensource/2020/12/15/introducing-cluster-api-provider-azure-capz-kubernetes-cluster-management/ by @craiglpeters and @jackfrancis.

**Comment by craiglpeters** [comment]
I'd love to hear about 1) your use case for cluster operations and/or 2) why CAPZ will/won't work for you

**Comment by kyuho24** [comment]
Hi craiglpeters

I try to use CAPZ. I did build test cluster according to the quick start guide (https://cluster-api.sigs.k8s.io/user/quick-start.html).
Then I want to check the auto-healing of CAPZ. First I simply deleted Egress rules from azure portal. CAPZ recovered it in 5 seconds. Then I deleted the azure resource group from azure portal(3d ago). Then Cluster didn't wake up. Only LB(x2), VNET, Public IP(x2), NSG(x2) and route were alive. I think there is some dependency problem and doesn't make resources in parallel. In activity log, azuer client account did't try to make VMs and only retry to make network resources. How can I fix this ?

clusterctl cli version : 0.3.10
Cluster API verison: v1alpha3
cluster (by CAPZ) version : 1.19.0
admin cluster : 1.18.4

**Reply by devigned** [reply]
@kyuho24, this is probably more suited to be an issue rather than discussion. Please consider opening an issue and we’ll root cause / resolve it. Cheers!
